job_executor/adapter/local_storage.py

```diff
@@ -409,10 +409,11 @@ def archive_input_files(dataset_name: str):
     Archives the input .tar files if not already archived
     """
     archive_dir = INPUT_DIR / "archive"
-    tar_file = INPUT_DIR / f"{dataset_name}.tar"
+    archived_tar_file = INPUT_DIR / f"archive/{dataset_name}.tar"
+    tar_file = INPUT_DIR / f"{dataset_name}.tar" 
```

Comment:
small suggestion couldn't help myself
```suggestion
    tar_filename = f"{dataset_name}.tar"
    archived_tar_file = archive_dir / tar_filename
    tar_file = INPUT_DIR / tar_filename
```